docs/user/user/adding-glean-to-your-project/javascript.md

```diff
@@ -8,15 +8,19 @@ Currently, these bindings support collecting data from [Browser Extensions](http
 
 ## Requirements
 
-* Node.js >= 12.20.0.
-* npm >= 7.0.0.
-* Webpack >= 5.34.0.
-* Python >= 3.6.[^1]
-* (_for browsers other than Firefox_) [webextension-polyfill](https://github.com/mozilla/webextension-polyfill) >= 0.8.0.[^2]
+* Node.js >= 12.20.0;
+* npm >= 7.0.0;
+* Webpack >= 5.34.0;
+* Python >= 3.6.
+  * The `glean` command requires Python to download [`glean_parser`](https://mozilla.github.io/glean_parser/) which is a Python library.
 
-[^1]: The `glean` command requires Python to download [`glean_parser`](https://mozilla.github.io/glean_parser/) which is a Python library.
+### Web extension specific requirements
 
-[^2]: Glean.js assumes a Promise-based `browser` API: Firefox provides such an API by default. Other browsers may require using a polyfill library such us `webextension-polyfill` when using Glean in browser extensions.
+* ["storage"](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions#api_permissions) API permission;
+* [Host permission](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions#host_permissions) to the Telemetry server;
+  * Only necessary if the defined server endpoint denies [cross-origin](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) requests.
```

Comment:
So it's not necessary for our endpoint? If so please call that out.

Reply:
It is necessary for our endpoint.

Reply:
Then maybe making this more explicit would be good